gwt-regexp-j2cl-tests/pom.xml

```diff
@@ -47,21 +33,15 @@
         </dependency>
         <dependency>
             <groupId>com.vertispan.j2cl</groupId>
-            <artifactId>gwttestcase-emul</artifactId>
```

Comment:
This project isn't using GWTTestCase at all, so removed it (and added the required junit sources)